src/main/java/ru/yandex/practicum/filmorate/controller/validation/ValidationExceptionHandler.java

```diff
@@ -0,0 +1,54 @@
+package ru.yandex.practicum.filmorate.controller.validation;
+
+import lombok.extern.slf4j.Slf4j;
+import org.jetbrains.annotations.NotNull;
+import org.springframework.http.HttpHeaders;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.MethodArgumentNotValidException;
+import org.springframework.web.bind.annotation.ControllerAdvice;
+import org.springframework.web.bind.annotation.ExceptionHandler;
+import org.springframework.web.context.request.WebRequest;
+import org.springframework.web.servlet.mvc.method.annotation.ResponseEntityExceptionHandler;
+
+/**
+ * обработчик исключений
+ */
+@Slf4j
+@ControllerAdvice
+public class ValidationExceptionHandler extends ResponseEntityExceptionHandler {
+    /**
+     * обработчик исключения согласно ТЗ
+      * @param ve исключение согласно ТЗ
+     * @param request Webrequest
+     * @return ResponseEntity
+     * @throws Exception исключение
+     */
+    @ExceptionHandler(ValidationException.class)
+        public ResponseEntity<Object> handleValidationException(ValidationException ve, WebRequest request) throws Exception {
+        log.warn(ve.getMessage());
```

Comment:
Лучше добавить в логирование какое-либо сообщение о том, что произошло, а с помощью заполнителя (placeholder)`{}`, добавить в сообщение лог ошибки.
Например:
```suggestion
        log.warn("Некорректно заполнены поля body: {}", ve.getMessage());
```

Reply:
исправлено